src/main/resources/assets/iris/lang/lol_us.json

```diff
@@ -1,33 +1,33 @@
 {
-  "iris.shaders.reloaded": "Shadews wewoaded!",
-  "iris.shaders.toggled": "Toggwed shadews to %s!",
-  "iris.keybind.reload": "Wewoad shadews",
-  "iris.keybind.shaderPackSelection": "Shadewpack sewection scween",
-  "iris.keybind.toggleShaders": "Toggwe Shadews",
-  "iris.keybinds": "Iwis",
-  "iris.shaders.reloaded.failure": "Faiwed to wewoad shadews! Reason: %s",
-  "iris.shaders.toggled.failure": "Faiwed to toggwe shadews! Reason: %s",
-  "iris.sodium.failure.title": "Iwis faiwed to woad!",
-  "iris.sodium.failure.reason": "You have Sodium in youw mods fowdew, howevew Iwis incwudes it. Pwease cwose da game, and wemove Sodium from youw mods fowdew.",
+  "iris.shaders.reloaded": "Shaedews wewoded!",
+  "iris.shaders.toggled": "Turnd Blek cat un grownd to %s!",
+  "iris.keybind.reload": "Rewoaded Blek cat un gownd",
+  "iris.keybind.shaderPackSelection": "Blek Ket On Grownd Bramd Selectur scren",
+  "iris.keybind.toggleShaders": "Toggwe Blek Cat Un Ground",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "COWLDNT REWOAD BLECK KET ON GROWND! Reson: %s",
+  "iris.shaders.toggled.failure": "COWLDNT FIND BLECK KET TO PUT UN GROWND! Reson: %s",
+  "iris.sodium.failure.title": "Iris cowld nut be able to woad!",
+  "iris.sodium.failure.reason": "You hab Sodium in kiten's mods fowder, however Iris includes it. Pleas cwose da game, and rembve Sodium from kitten's mods fowder.",
```

Comment:
This translation is outdated, replace with the `notFound` and `incompatible` translations
"Iris needz Sodium fr gud purrformans, but Sodium iznt heere. Plz downlod Sodium, put it in yr mods foldur, n relawnch da game."
"Iris needz Sodium fr gud purrformans, but yr vershun iznt compatibul. Plz downlod Sodium, put it in yr mods foldur, n relawnch da game."

Reply:
lol